util/execdetails/execdetails.go

```diff
@@ -381,13 +383,19 @@ func (e *RuntimeStats) Record(d time.Duration, rowNum int) {
 	atomic.AddInt32(&e.loop, 1)
 	atomic.AddInt64(&e.consume, int64(d))
 	atomic.AddInt64(&e.rows, int64(rowNum))
+	atomic.AddInt64(&e.concurrency, 0)
 }
 
 // SetRowNum sets the row num.
 func (e *RuntimeStats) SetRowNum(rowNum int64) {
 	atomic.StoreInt64(&e.rows, rowNum)
 }
 
+// SetConcurrencyNum sets the concurrency num.
+func (e *RuntimeStats) SetConcurrencyNum(concurrencyNum int64) {
+	atomic.StoreInt64(&e.concurrency, concurrencyNum)
+}
+
 func (e *RuntimeStats) String() string {
-	return fmt.Sprintf("time:%v, loops:%d, rows:%d", time.Duration(e.consume), e.loop, e.rows)
+	return fmt.Sprintf("time:%v, loops:%d, rows:%d, concurrency:%d", time.Duration(e.consume), e.loop, e.rows, e.concurrency)
```

Comment:
It's better display concurrency details for each operator. For example, in hash aggregate operator, we can show how many partial and final workers respectively.

Reply:
Update. PTAL